interface/src/app/signature/resources.js

```diff
@@ -11,6 +11,12 @@ angular.module('adage.signature.resources', [
   }
 ])
 
+.factory('ParticipationType', ['$resource', 'ApiBasePath',
+  function($resource, ApiBasePath) {
+    return $resource(ApiBasePath + 'participationtype/');
```

Comment:
Can you remove the trailing `/` in the API so that it will be consistent with others?